SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -1834,6 +1834,12 @@ class CVariable {
    */
   inline virtual void SetVelocity_Old(unsigned long iPoint, const su2double *val_velocity) {}
 
+  /*!
+   * \brief A virtual member.
+   * \param[in] val_energy - Pointer to energy.
+   */
+  inline virtual void SetEnergy_Old(unsigned long iPoint, vector<su2double>& val_energy) {}
```

Comment:
Looks like this is not used.

Reply:
This should not be on this commit, my bad. But it will be used in future commits for CNEMONSSolver::BC_Smoluchowski_Maxwell.

Reply:
Ok, for when the time comes, if the function is only used in CNEMOEuler/NSSolver, it does not need to be declared in the base CVariable, just in CNEMOEuler/NSVariable.